bindings/python/setup.py

```diff
@@ -30,6 +30,8 @@
         "Programming Language :: Python :: 3.6",
         "Programming Language :: Python :: 3.7",
         "Programming Language :: Python :: 3.8",
+        "Programming Language :: Python :: 3.9",
```

Comment:
already supports python3.9 but missing this [classifier](https://pypi.org/classifiers/)